creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -318,8 +338,8 @@ class SnapbackSM {
       },
       defaultJobOptions: {
         // removeOnComplete is required since the completed jobs data set will grow infinitely until memory exhaustion
-        removeOnComplete: true,
-        removeOnFail: true
+        removeOnComplete: SNAPBACK_QUEUE_HISTORY,
+        removeOnFail: SNAPBACK_QUEUE_HISTORY
```

Comment:
do we want to keep this many jobs? I wonder if this will cause us to store too much data and potentially oom. also, is there also a way to keep jobs based on a time basis?

Reply:
there is a way to keep based on time, but 500 has a very low impact on memory since only a few fields are stored (jobId, start time, etc) and no job logs or anything like that. also this was on staging for days with healthy redis memory usage